### PR TITLE
Remove unnecessary tool-tips and appset specific details

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**

### CODE REVIEW

File: packages/mco/components/mco-dashboard/data-policy/summary-card/summary-card.tsx [hunk 1]
```diff
@@ -107,13 +105,6 @@ export const SummaryCard: React.FC = () => {
             <GalleryItem className="mco-dashboard__contentColumn">
               <Text className="text-muted mco-dashboard__statusText--margin mco-dashboard__statusText--size">
                 {t('Applications')}
-                <FieldLevelHelp>
-                  <Trans t={t}>
-                    The applications count displays the total number of{' '}
-                    <b>ApplicationSet</b> type applications in all disaster
-                    recovery configured clusters.
-                  </Trans>
-                </FieldLevelHelp>
```
Comment (GowthamShanmugam) [inline packages/mco/components/mco-dashboard/data-policy/summary-card/summary-card.tsx:116]: We should not remove this toolTip completely. Instead, It should clearly mention like:
```
The applications count displays the total number of applications in all disaster recovery configured clusters.
```

File: packages/mco/components/mco-dashboard/data-policy/summary-card/summary-card.tsx [hunk 2]
```diff
@@ -107,13 +105,6 @@ export const SummaryCard: React.FC = () => {
             <GalleryItem className="mco-dashboard__contentColumn">
               <Text className="text-muted mco-dashboard__statusText--margin mco-dashboard__statusText--size">
                 {t('Applications')}
-                <FieldLevelHelp>
```
Comment (GowthamShanmugam) [inline packages/mco/components/mco-dashboard/data-policy/summary-card/summary-card.tsx:110]: you missed volume replication pie chart text change: Volume replication health (ApplicationSet)

File: packages/mco/components/mco-dashboard/data-policy/summary-card/summary-card.tsx (hunk 1 above)
Comment (GowthamShanmugam) [reply packages/mco/components/mco-dashboard/data-policy/summary-card/summary-card.tsx:116]: Check with UX about these two options

Comment (Chandan141996) [reply packages/mco/components/mco-dashboard/data-policy/summary-card/summary-card.tsx:116]: Better to go with option 2 for now to be consistent with alert popover text

**Comment by TimothyAsirJeyasing** [comment]
image

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**